### PR TITLE
ome-www deployment updates

### PR DESCRIPTION
**Pull request by kennethgillen**
updates to resize the LVM lvs and filesystems
also configured jekyll-build with a site name

### CODE REVIEW

**Comment by kennethgillen** [comment]
Supporting config required: https://github.com/openmicroscopy/management_tools/pull/492

File: ansible/server-state-playbooks/www/requirements.yml [hunk 1]
```diff
@@ -16,9 +13,6 @@
   src: https://github.com/openmicroscopy/ansible-role-jekyll-build.git
   version: 1.2.0
 
-- name: openmicroscopy.lvm-partition
```
Comment (sbesson) [inline ansible/server-state-playbooks/www/requirements.yml]: Is `lvm-partition` pulled as a dependency of another role? What happens if a deployment includes a version < 1.1.0?

File: ansible/server-state-playbooks/www/www.yml [hunk 2]
```diff
@@ -58,6 +84,7 @@
       jekyll_build_git_repo: "https://github.com/openmicroscopy/www.openmicroscopy.org"
       jekyll_build_force_rebuild: True
       jekyll_build_config: ['_config.yml', '_prod.yml']
+      jekyll_build_name: www.openmicroscopy.org
```
Comment (sbesson) [inline ansible/server-state-playbooks/www/www.yml]: One more commit commit to reflect this change in https://github.com/openmicroscopy/infrastructure/blob/master/ansible/server-state-playbooks/www/www.yml#L212

File: ansible/server-state-playbooks/www/requirements.yml (hunk 1 above)
Comment (kennethgillen) [inline ansible/server-state-playbooks/www/requirements.yml]: It won't be used by a sub-role I'd have thought, given it makes system-wide changes. Certainly not here.

**Comment by sbesson** [comment]
Deployed successfully with @kennethgillen. Overall :+1: for the changes and using non-default deployment locations. Merging as the state of the live deployment.

File: ansible/server-state-playbooks/www/requirements.yml (hunk 1 above)
Comment (manics) [inline ansible/server-state-playbooks/www/requirements.yml]: It's used by the docker role if you choose the custom storage option https://github.com/openmicroscopy/ansible-role-docker#custom-storage

**Pull request merged**